src/Storages/ObjectStorage/Azure/Configuration.cpp

```diff
@@ -291,8 +292,82 @@ void StorageAzureConfiguration::fromDisk(const String & disk_name, ASTs & args,
         structure = *parsing_result.structure;
 }
 
+void StorageAzureConfiguration::initializeForOneLake(ASTs & args, ContextPtr context)
```

Comment:
Maybe we can combine this function with `fromAST` function. Processing of abfss url is same, atleast we can put that in one function and use. What do you think?